### DISCUSSION TITLE
[Feature] List external libraries under the "Library" section on the web and mobile interfaces

### DISCUSSION THREAD

**Discussion by troycarpenter** [immich-app/immich#4622] [Feature Request]
### The feature

When external libraries are in use, the photos all end up on the timeline (like all other albums, no surprise there). However, it would be nice to have entries for the external libraries on the web GUI and in the mobile apps in the appropriate places.

This link could (and probably would be) a quick search filter.

However, if there is an easy way to search for photos that are in external libraries, please let me know.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by Joshsg02** [comment]
Same here. This helps me organize and separate my phone's photos from existing holiday & family photos and allows me to share them with family and friends. I imagine other operate similarly to this.

Hopefully this feature gets an official response.
